scripts/release/generate_history_notes.py

```diff
@@ -115,7 +116,13 @@ def modify_history_file(file: fileinput.FileInput, new_history: str):
 
 def construct_cli_history(component: str):
     history = '**{}**\n\n'.format(component)
+    non_breaking_change_notes = []
     for note in history_notes[component]:
+        if 'BREAKING CHANGE' in note.upper():
+            history += '* {}\n'.format(note)
```

Comment:
`str.join` is considered better than string concatenation because it is much faster and more Pythonic. See
- Guido van Rossum' essay: https://www.python.org/doc/essays/list2str/
- https://stackoverflow.com/questions/3055477/how-slow-is-pythons-string-concatenation-vs-str-join
- https://towardsdatascience.com/do-not-use-to-join-strings-in-python-f89908307273

Reply:
Updated.